tests/subsys/settings/functional/fcb/boards/particle_xenon.conf

```diff
@@ -5,4 +5,3 @@
 #
 CONFIG_SPI=y
 CONFIG_SPI_NOR=y
-CONFIG_SPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
```

Comment:
It is not so simple that you can change the layout with no harm to subsystems. For instance this change imply that settings will use fcb witch sectors of size of 256 B instead of 4096 B.
Similar note is up to littlefs.

What is needed is the a way for configure the size via DTS for SFDP_MINIMAL and/or fill of DTS SFDP records where it is missing.

Reply:
Hum, it is ture that this change has side effect with big change, I rework a new PR just fix some mirror bugs when I countered.

Reply:
> page layout member page_size is sector size?

Yes. For memories devices which distinguish the `block size`, the `sector size` and the `page size` it might be decided to configure SPI_NOR_FLASH_LAYOUT_PAGE_SIZE to a multiplication of any of them.

Reply:
I think it is a confused variable without any note